featuretools/primitives/utils.py

```diff
@@ -93,6 +97,51 @@ def list_primitives():
     return pd.concat([agg_df, transform_df], ignore_index=True)[columns]
 
 
+def summarize_primitives() -> pd.DataFrame:
+    """Returns a metrics summary DataFrame of all built-in primitives in featuretools."""
```

Comment:
"Returns a metrics summary DataFrame of all primitives in featuretools.primitives." would be more accurate.  ``nlp_primitives`` and other libraries that add primitives to `featuretools.primitives` will be included.

Reply:
or "all primitives found in list_primitives"